kafka/consumer.go

```diff
@@ -228,6 +231,7 @@ func (c *Consumer) fetch(ctx context.Context) error {
 			c.cfg.Logger.Error("unable to process event",
 				zap.Error(err),
 				zap.String("topic", msg.Topic),
+				zap.ByteString("message.value", msg.Value),
```

Comment:
**question**: logging of headers and message values seem to differ wildly. When should they be logged ? Does it make sense to always log them ?

Reply:
I added the message value in the log above since that error occurs on decoding a message. Processing the batch may happen for any reason and it seemed to me that it wasn't necessary to log the raw `message.value`. I still think it's best to not log the message.value wherever possible.

Reply:
Oh I see, I've updated the code to remove the message.value during processing errors.

Should the headers also be removed ? Are they valuable ?

Reply:
I think it's fine to log the headers, and may be useful to be able to identify failing patterns for given metadata.